services/net/clip/ClipAction.cs

```diff
@@ -85,7 +85,6 @@ public ClipAction(IApiService api, IOptions<ClipOptions> options, ILogger<ClipAc
                     {
                         // If another process has it in progress only attempt to do an import if it's
                         // more than an 2 minutes old. Assumption is that it is stuck.
-                        reference = await this.UpdateContentReferenceAsync(reference, WorkflowStatus.InProgress);
```

Comment:
I don't understand what you are doing here?  The reason this code is here is to act as a locking mechanism so that other processes don't attempt to work with the same content at the same time.

Reply:
Updated, initially thought this could be redundant as we just retrieve the reference on line 78